validator/sawtooth_validator/gossip/gossip.py

```diff
@@ -434,6 +455,10 @@ def add_candidate_peer_endpoints(self, peer_endpoints):
     def set_connection_status(self, connection_id, status):
         self._connection_statuses[connection_id] = status
 
+    def remove_endpoint(self, endpoint):
+        if endpoint in self._endpoints_to_peer:
+            del self._temp_endpoint[endpoint]
```

Comment:
Should this be:
```
if endpoint in self._temp_endpoints:
    del self._temp_endpoints[endpoint] # <-- add s
```

Reply:
Also, should it remove the peer from `self._peers`?

Reply:
No to self._peers, it will not be in temp_endpoints if it has made it to peers. This is mainly to clean up endpoints that do not make it through authorization.

Reply:
Changed name of function to remove_temp_endpoint to make this clear.